server/schema/dashboard.go

```diff
@@ -3,9 +3,11 @@ package schema
 import (
 	"database/sql/driver"
 	"fmt"
+	"reflect"
```

Comment:
I use go imports for vim, so it's not my idea to put it here
Moreover
https://github.com/TheQuestionru/test-golang/pull/11/files/a8053384faaba02f56d75914c4380992ebbca7f9#diff-702aefdd0bcda74c00b87c7887175149L8
https://github.com/TheQuestionru/test-golang/pull/11/files/a8053384faaba02f56d75914c4380992ebbca7f9#diff-702aefdd0bcda74c00b87c7887175149R106